### DISCUSSION TITLE
cli and server does not match version, how to fix it?

### DISCUSSION THREAD

**Discussion by minzak** [hasura/graphql-engine#9636] [Question] [answered]
When I run hasura 
```
cd project
/usr/local/bin/hasura --no-color --log-level debug console --no-browser --address 127.0.0.1 --console-port 9695 --api-port 9693 --console-hge-endpoint http://127.0.0.1:8090

```
I got some messages and present these strings:
```
level=debug msg="versions: cli: [v2.24.1] server: [v2.4.1]"
level=debug msg="compatibility check: [true] cli and server does not match"
```

But cli v2.4.1 is absent in repository..
How to build all with the same version?
Thanks.

Version file:
```
root@graphql:~# hasura version
INFO hasura cli                                    version=v2.24.1
root@graphql:~# graphql-engine version
Hasura GraphQL Engine: v2.4.1
```

**Comment by minzak** [comment]
Not sure but similar to https://github.com/hasura/graphql-engine/discussions/9302

**Reply by meetzaveri** [reply, marked as answer]
Hey @minzak You should update graphql-engine version to `v2.24.1` . Relevant link to graphql-engine image distribution on docker hub - https://hub.docker.com/r/hasura/graphql-engine/tags?page=1&name=2.24.1

There isn't image distribution for hasura graphql-engine `2.4.1` . For reference, you can check this link as it will not show any results - https://hub.docker.com/r/hasura/graphql-engine/tags?page=1&name=2.4.1
